tests/ai_pipeline/test_pipeline.py

```
@@ -0,0 +1,368 @@
+def _make_unet_output(disc_val: float = 0.9, cup_val: float = 0.7) -> torch.Tensor:
+    output = torch.zeros(1, 2, 512, 512)
+    output[0, 0] = disc_val
+    output[0, 1] = cup_val
+    return output
+
+
+# ---------------------------------------------------------------------------
+# Fixtures
+# ---------------------------------------------------------------------------
+
+@pytest.fixture()
```

Ruff (0.15.12)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)